services/nomad/build/buildbot.nomad

```diff
@@ -0,0 +1,164 @@
+job "buildbot" {
+  type = "service"
+  // TODO
+  // datacenters = ["VOID"]
+  // namespace = "build"
+  datacenters = ["dc1"]
+
+  group "buildbot" {
+    network {
+      mode = "bridge"
+      hostname = "void-buildbot"
+
+      // TODO: static only for testing
+      port "http" { static = 8010 }
+
+      // TODO: add when deploying
+      port "worker" { /* host_network = "internal" */ }
+    }
+
+    volume "buildbot_database" {
+      type = "host"
+      source = "buildbot_database"
+      read_only = false
+    }
+
+    service {
+      // TODO: can't be nomad with meta {}
+      provider = "nomad"
+      name = "buildbot-www"
+      port = "http"
+
+      // TODO
+      // meta {
+      //   nginx_enable = "true"
+      //   nginx_names = "build.voidlinux.org"
+      // }
+
+      check {
+        type = "http"
+        address_mode = "host"
+        path = "/"
+        timeout = "30s"
+        interval = "5s"
+      }
+    }
+
+    service {
+      provider = "nomad"
+      name = "buildbot-worker"
+      port = "worker"
+    }
+
+    task "buildbot" {
+      driver = "docker"
+
+      config {
+        image = "localhost:5000/infra-buildbot:latest"
+        force_pull = true
+        ports = ["http", "worker"]
+      }
+
+      volume_mount {
+        volume = "buildbot_database"
+        destination = "/db"
+      }
+
+      template {
+        data = <<EOF
+[buildbot]
+worker-port = {{ env "NOMAD_PORT_worker" }}
+www-port = {{ env "NOMAD_PORT_http" }}
+# TODO
+#url = http://build.voidlinux.org/
+
+[irc]
+host = irc.libera.chat
+# TODO
+# nick = void-builder
+# channel = #xbps
+# authz = duncaen maldridge
+nick = abby-testbot
+channel = ###abby-testbot
+authz = duncaen maldridge abby
+EOF
+        destination = "local/config.ini"
+      }
+
+      template {
+        data = <<EOF
```

Comment:
This should be built out using `jsonencode()`.  Doing this in a heredoc breaks syntax highlighting among other things.